packages/p2p/stores/general-store.js

```diff
@@ -357,8 +369,13 @@ export default class GeneralStore {
         if (!response.error) {
             this.setAdvertiserId(p2p_advertiser_info.id);
             this.setIsAdvertiser(!!p2p_advertiser_info.is_approved);
+            this.setIsBarred(!!p2p_advertiser_info.blocked_until);
```

Comment:
Can be `computed`, no need to create an extra observable here.

Reply:
This is being set from the response and the observable is being used as a check in the computed `blocked_until_date_time`. I'm not sure how it can be a computed.

Reply:
I'm not sure how this will be an observable.

Reply:
You could use `user_blocked_until` for this. e.g.

```JS
@computed
get user_is_barred() {
    return !!this.user_blocked_until;
}
```

Reply:
You're right, good idea.